Add frontend and backend part for job to materialize a volume annotation (#6086)
scalableminds/webknossos: docs/volume_annotation.md

The active segment ID under the cursor can be found in the status bar at the bottom of the screen or through the context-sensitive menu on right-click.

### Merging volume annotation with fallback data

> Once finished annotating a volume layer with a fallback layer, the state of this volume layer can be materialized into a new dataset. for this go to the layer settings in the left border. on the top right of the volume layer is the following button:

Reviewer: ```suggestion
After finishing the annotation of a volume layer with a fallback layer, the combined state of these layers can be materialized into a new dataset. For this go to the layer settings in the left border tab. On the top right of the volume layer is the following button:
```

> After finishing the annotation of a volume layer with a fallback layer, the combined state of these layers can be materialized into a new dataset. For this go to the layer settings in the left border tab. On the top right of the volume layer is the following button:

Reviewer 2: ```suggestion
After finishing the annotation of a volume layer with a fallback layer, the combined state of these layers can be materialized into a new dataset. For this, go to the layer settings in the left border tab. On the top right of the volume layer is the following button:
```

![Video: Proof-Reading Volume Annotations](https://www.youtube.com/watch?v=Sq4AuWanK14)

> After finishing the proof-reading a long-running job can be started to apply the combining of segments into a new dataset with the same layers. The job can be started via a modal. The modal can be open by clicking on the button next to the merger mode button.

Reviewer 2: ```suggestion
After finishing the proof-reading, a long-running job can be started to apply the merging of segments into a new dataset with the same layers. The job can be started via a modal. The modal can be open by clicking on the button next to the merger mode button.
```

![Video: Proof-Reading Volume Annotations](https://www.youtube.com/watch?v=Sq4AuWanK14)

> After finishing the proof-reading, a long-running job can be started to apply the merging of segments into a new dataset with the same layers. The job can be started via a modal. The modal can be open by clicking on the button next to the merger mode button.

Reviewer: ```suggestion
After finishing the proof-reading, a long-running job can be started to apply the merging of segments into a new dataset with the same layers. The job can be started via a modal. The modal can be opened by clicking on the button next to the merger mode button.
```
